js/BranchingSet.js

```diff
@@ -308,6 +308,8 @@ export default class BranchingSet {
           parentView.nthChild--;
         }
       }
+      const descendants = model.getAllDescendantModels();
+      descendants.reverse().forEach(model => data.remove(model));
```

Comment:
Minor: The 'model' variable is used in the upper scope already
```suggestion
      descendants.reverse().forEach(descendantModel => data.remove(descendantModel));
```

Reply:
It shouldn't be a problem here, as the variables aren't hoisted into the parent function because they are defined in the block scope of the fat-arrow function, its possible to use the same variable name in multiple nested function scopes without assuming they're related.

It's equivalent to using:
```js
const model = null;

descendants.reverse().forEach(function (model) {
  // this is a different model variable in each iteration
  data.remove(model)
});
```

Reply:
Yeah I know there isn't an issue, just that sometimes they will get flagged up by linters etc